index.js

```
@@ -5,6 +5,7 @@ module.exports = {
     node: true,
     es6: true
   },
+  extends: ["eslint:recommended", "plugin:react/recommended", "plugin:react-hooks/recommended"],
```

We don't currently have react-hooks as a peer dep. Does this fail silently or should we add that dep explicitly?

---

I was actually thinking about that. I'm not really sure what the behavior is but it's probably safest to add it?

---

I think so -- all our downstream projects will already use it (or want to use it).